src/CustomComponentOptions.ts

```diff
@@ -19,6 +19,11 @@ export interface CustomComponentOptions {
      */
     allowedParentTags: string[]
 
+    /**
+     * Allowed tags for child
+     */
+    allowedChildTags: string[]
```

Comment:
fix: `allowedChildTags: string[]` => `allowedChildTags?: string[]`

This way the attribute is not required and not a breaking change